apps/site/assets/ts/schedule/components/ScheduleDirection.tsx

```diff
@@ -205,6 +205,20 @@ const ScheduleDirection = ({
           <ScheduleDirectionButton dispatch={dispatch} />
         ) : null}
       </div>
+      {route.type === 1 || route.type === 0
```

Comment:
suggestion (non-blocking): This could be pulled out into a helper function, maybe called something like `isASubwayRoute` or something along those lines. This would 1) consolidate repeated logic, 2) describe the point of the logic through the function name, and 3) be testable.